syft/workers/base.py

```diff
@@ -175,7 +180,7 @@ def send_msg(self, msg_type, message, location):
 
         return response
 
-    def recv_msg(self, bin_message):
+    def recv_msg(self, bin_message: bin) -> bin:
```

Comment:
Like _recv_msg this does not return binary data

Reply:
@robert-wagner The doc string for this method `recv_msg` (notice no underscore at the beginning of the method name) actually says:

`Returns:
A binary message response
`

Is that an error?

Reply:
@robert-wagner The doc string for this method `recv_msg` (notice no underscore at the beginning of the method name) actually says:

`Returns:
A binary message response
`

Is that an error?

Reply:
@robert-wagner The doc string for this method `recv_msg`  actually says:

`Returns:
A binary message response
`

Is that an error?

Reply:
Nevermind you are correct